CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * Fixed `RSpec/ContextWording` missing `context`s with metadata. ([@pirj][])
 * Fix `FactoryBot/AttributeDefinedStatically` not working with an explicit receiver. ([@composerinteralia][])
 * Add `RSpec/Dialect` enforces custom RSpec dialects. ([@gsamokovarov][])
+* Add 'for' and 'if' default prefixes for `ContextWording`. ([@josephan][])
```

Comment:
Hmm, for some reason it's not auto-linking to your GitHub profile, any idea why?

Reply:
@josephan needs to be added to the bottom of CHANGELOG.md. These are not links that github populates, these are references to the links in the contributors section at the bottom.

Reply:
Done!